common/lib/app.js

```diff
@@ -58,51 +58,58 @@
   };
 
   bgApp.sendNotification = function (streamList){
-    var displayCount = 5;
-    var opt;
-    var defaultIcon = utils.runtime.getURL("common/icons/64_2.png");
+    var displayCount = settings.get("notifyCount").get("value");
+    var defaultIcon = utils.runtime.getURL("common/icons/64_1.png");
     var streamsToShow = streamList.slice(0, displayCount);
-    var isSingle = streamsToShow.length === 1;
-    var streamTitles = streamsToShow.map(function (c){
+    var streamsOther = streamList.slice(displayCount, streamList.length);
+    var streamTitles = streamsOther.map(function (c){
       return c.get("channel").display_name;
     });
 
     if ( bgApp.richNotificationsSupported() ) {
-
-      var notificationId = _.uniqueId("TwitchNow.Notification.");
-
-      try {
-        if ( isSingle ) {
-          var singlePreview = streamsToShow[0].get("preview") && streamsToShow[0].get("preview").medium;
-          opt = {
-            type   : "basic",
-            title  : streamsToShow[0].get("channel").display_name,
-            message: streamsToShow[0].get("channel").game + "\n" + streamsToShow[0].get("channel").status,
-            iconUrl: streamsToShow[0].get("channel").logo
+      let sec = 0;
+      for (let i in streamsToShow) {
+        try {
+          let num = i;
+          let notificationId = _.uniqueId("TwitchNow.Notification.");
+          let opt = {
+              type    : "basic",
+              title   : (streamsToShow[num].get("stream_type") != 'live' ? '🔄 ' : '') + streamsToShow[num].get("channel").display_name,
+              message : streamsToShow[num].get("channel").game + "\n" + streamsToShow[num].get("channel").status,
+              iconUrl : streamsToShow[num].get("channel").logo
           }
-
-          bgApp.notificationIds[notificationId] = streamsToShow[0];
-
-        } else {
-          opt = {
-            type   : "basic",
-            title  : "Twitch Now",
-            message: streamTitles.join("\n"),
-            iconUrl: defaultIcon
+          bgApp.notificationIds[notificationId] = streamsToShow[num];
+          setTimeout(function() {
+              chrome.notifications.create(notificationId, opt, function (){});
+          }, sec);
+          sec = (sec + 1000)
+          if (i == (displayCount - 1) && streamsOther.length != 0) {
```

Comment:
Здесь обход массива streamsToShow можно закончить и выкинуть условие на проверку i
```suggestion
}
          if (streamsOther.length) {
```

Reply:
Тогда надо и за пределы try catch вынести.

Reply:
да нужно